src/EnergyPlus/MixedAir.cc

```diff
@@ -4593,17 +4718,34 @@ namespace MixedAir {
 
 		if ( OAController( OAControllerNum ).ControllerType_Num == ControllerOutsideAir ) {
 			// The outside air controller sets the outside air flow rate and the relief air flow rate
-			Node( OutAirNodeNum ).MassFlowRate = OAController( OAControllerNum ).OAMassFlow;
-			Node( InletAirNodeNum ).MassFlowRate = OAController( OAControllerNum ).OAMassFlow;
-			Node( RelAirNodeNum ).MassFlowRate = OAController( OAControllerNum ).RelMassFlow;
-			Node( OutAirNodeNum ).MassFlowRateMaxAvail = OAController( OAControllerNum ).OAMassFlow;
+			if (OAController(OAControllerNum).ManageDemand)
+			{
+				Node(OutAirNodeNum).MassFlowRate = OAController(OAControllerNum).DemandLimitFlowRate;
+				Node(InletAirNodeNum).MassFlowRate = OAController(OAControllerNum).DemandLimitFlowRate;
+				Node(OutAirNodeNum).MassFlowRateMaxAvail = OAController(OAControllerNum).DemandLimitFlowRate;
+			}
+			else
+			{
+				Node(OutAirNodeNum).MassFlowRate = OAController(OAControllerNum).OAMassFlow;
+				Node(InletAirNodeNum).MassFlowRate = OAController(OAControllerNum).OAMassFlow;
```

Comment:
This section of modification can be avoided, because "OAController( OAControllerNum ).OAMassFlow" was set as "OAController(OAControllerNum).DemandLimitFlowRate" before at Line 4272. 

Let's minimize the modifications of the original codes.